ion/tokenizer.go

```diff
@@ -1263,3 +1273,24 @@ func (t *tokenizer) unread(c int) {
 	t.pos--
 	t.buffer = append(t.buffer, c)
 }
+
+func isInvalidChar(c int) bool {
```

Comment:
This name's a bit ambiguous at the top level. Maybe `isInvalidStringChar`? It'd fit nicely alongside the other `isXxx` functions in textutils.go.

Reply:
Now that's embarrassing! I read [this comment](https://github.com/amzn/ion-go/pull/58#discussion_r436227420) and thought you're suggesting to change the name of the function, totally misread it.
I originally had named it ```isProhibitedControlChar```. Changing it back